projects/adrv9009/src/devices/adi_hal/parameters.h

```diff
@@ -102,41 +102,102 @@
 #else
 #define GPIO_DEVICE_ID			XPAR_PS7_GPIO_0_DEVICE_ID
 #endif
-#ifdef XPS_BOARD_ZCU102
```

Comment:
is the GPIO_OFFSET no longer used ?

Reply:
it seems unrelated with this commit

Reply:
GPIO_OFFSET will likely disappear from all projects when it will be implemented in gpio driver.